easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.3-2-goolf-1.4.10.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '6.9.3-2'
+
+homepage = 'http://www.imagemagick.org/'
+description = """ImageMagick is a software suite to create, edit, compose, or convert bitmap images"""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.imagemagick.org/download/']
```

Comment:
@pescobar: please change to looks like this is a good alternative: `https://launchpad.net/imagemagick/main/%(version)s/+download/`, where all versions are available